tests/pyfunc/test_spark.py

```diff
@@ -1234,22 +1255,39 @@ def predict(self, context, model_input):
     assert res["res"][0] == ("string")
 
 
-def test_spark_udf_return_nullable_array_field(spark):
+def test_spark_udf_with_model_serving(spark):
     class TestModel(PythonModel):
-        def predict(self, context, model_input):
-            values = [np.array([1.0, np.nan])] * (len(model_input) - 2) + [None, np.nan]
-            return pd.DataFrame({"a": values})
+        def predict(self, context, model_input, params=None):
+            return ["string"] * len(model_input)
 
-    with mlflow.start_run():
-        mlflow_info = mlflow.pyfunc.log_model(
+    test_params = {
+        "str_param": "str_a",
```

Comment:
Is this param used in the python model ?

Reply:
It's passed as params, but not in the output though.